app/scripts/controllers/permissions/index.js

```diff
@@ -434,6 +434,8 @@ export class PermissionsController {
           name: CAVEAT_NAMES.exposedAccounts,
         },
       )
+
+      ethAccounts.caveats.push({ type: 'limitResponse', value: 1, name: CAVEAT_NAMES.primaryAccountOnly })
```

Comment:
This should be `limitResponseLength`, right?

Reply:
shit, ctrl-z'd too much

Reply:
fwiw my comment about testing this locally was before "cleaning up my console.logs" which apparently included undoing "updating my stash from before the limitResponse->limitResponseLength feedback"